.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## This is how are pull requests handled by FreeRDP
+1. Every new pull request needs to build and pass the unit tests at https://ci.freerdp.com
+1. At least 1 (better two) people need to review and test a pull request and agree to accept
+
+## Preparations before creating a pull
+* Rebase your branch to current master
+* Try to clean up your commit history, group changes to commits
+* Check your formatting! A _astyle_ script can be found at ```./scripts/format_code.sh```
+* Run a clang scanbuild before and after your changes to avoid introducing new bugs
+* Run your compiler at pedantic level to check for new warnings
+
+## To ease accepting your contribution
+* Give your pull request a proper name and description so people looking at it have an idea what it is for
```

Comment:
Maybe split that:
* Give the pull request a proper name so people looking at it have an basic idea what it is for
* Add at least a brief description what it does (or should do :) and what it's good for